lib/mergometer/pull_request.rb

```diff
@@ -82,40 +88,52 @@ def approval_time
       (first_approval.submitted_at - created_at).in_hours
     end
 
+    def review_required?
+      number_of_given_reviews == 0 && open? && !wip?
```

Comment:
What do you think of extracting `number_of_given_reviews == 0` to a private method? (Eg. `no_reviews?`.)